src/main/java/randoop/compile/SequenceCompiler.java

```diff
@@ -173,6 +173,15 @@ private boolean compile(
         compiler.getTask(
             null, fileManager, diagnostics, new ArrayList<String>(compilerOptions), null, sources);
     Boolean succeeded = task.call();
+
+    //    // Print the diagnostics if compilation failed
```

Comment:
Please do one of:
 * Make this unconditional
 * Put this behind an "if" statement.
 * Delete this.